### PR TITLE
Issue/120/create collected points funcionality

### PR DESCRIPTION
**Pull request by Bedpaw**

### CODE REVIEW

File: src/components/organisms/collected-points.vue [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
-  },
+
   data: () => ({
+    pointCategories: [
+      {
+        name: 'Pierwsza kategoria',
+        categoryId: 1,
+        pointValue: 1,
+        pointShape: 1,
+        imageColor: 'f-first_category',
```
Comment (Bedpaw) [inline src/components/organisms/collected-points.vue:30]: first-category

File: src/style/variables/colors/dark.sass [hunk 2]
```diff
@@ -14,5 +14,11 @@ $color-info: palette.$Cerulean
 $color-warn: palette.$Orange
 $color-error: palette.$Red
 
+$color-shape-1: palette.$Purple
+$color-shape-2: palette.$Red
```
Comment (Bedpaw) [inline src/style/variables/colors/dark.sass:18]: to fix

File: src/components/molecules/table-row/collected-points.vue [hunk 3]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <m-table-row :has-details="true" add-class="f-collected-points">
+    <template v-slot:cols>
+      <div>
+        <icon-star :size="64" class="a-icon" :class="category.imageColor"/>
+      </div>
+
+      <div class="f-text-32 f-text-bold">
+        {{quantity}}
```
Comment (Drego31) [inline src/components/molecules/table-row/collected-points.vue:9]: Here it should be like that `{{ quantity }}` with spaces

File: src/components/molecules/table-row/collected-points.vue [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      </div>
+
+      <div class="f-text-32 f-text-bold">
+        {{value}}
```
Comment (Drego31) [inline src/components/molecules/table-row/collected-points.vue:13]: Like above

Comment (Bedpaw) [reply src/components/molecules/table-row/collected-points.vue]: done

**Pull request merged**